pkgs/applications/video/mpv/default.nix

```diff
@@ -93,6 +93,13 @@ in stdenv.mkDerivation rec {
     sha256 = "0i2nl65diqsjyz28dj07h6d8gq6ix72ysfm0nhs8514hqccaihs3";
   };
 
+  # FIXME: Remove this patch for building on macOS if it gets released in
+  # the future.
+  patches = optional stdenv.isDarwin (fetchpatch {
+    url = https://patch-diff.githubusercontent.com/raw/mpv-player/mpv/pull/6040.patch;
```

Comment:
IIRC this is not a good url to use. For instance if you rebase or add commits the sha256 will change. Instead , you should refer to the commit rev like:

https://github.com/mpv-player/mpv/commit/dc553c8cf4349b2ab5d2a373ad2fac8bdd229ebb.patch